docs/src/pages/power-profiler.js

```diff
@@ -207,6 +223,68 @@ function PowerProfiler() {
                   </>
                 )}
               </div>
+              <div className="col col--4">
+                <div className="profilerInput">
+                  <label>Backlight</label>
+                  <input
+                    checked={backlightEnabled}
+                    id="backlight"
+                    {...bindBacklightEnabled}
+                    className="toggleInput"
+                    type="checkbox"
+                  />
+                  <label htmlFor="backlight" className="toggle">
+                    <div className="toggleThumb" />
+                  </label>
+                </div>
+                {backlightEnabled && (
+                  <>
+                    <div className="profilerInput">
+                      <label>LED Quantity</label>
+                      <div className="inputBox">
+                        <input {...bindBacklightQuantity} type="number" />
+                      </div>
+                    </div>
+                    <div className="profilerInput">
+                      <label>LED Voltage</label>
```

Comment:
```suggestion
                      <label>LED Forward Voltage</label>
```
Let's be explicit here